src/main/java/org/opentripplanner/gtfs/mapping/StopMapper.java

```diff
@@ -39,6 +45,27 @@ private Stop doMap(org.onebusaway.gtfs.model.Stop gtfsStop) {
         gtfsStop.getId().getAgencyId()
     );
 
+    if (translationHelper != null) {
+      return new Stop(base.getId(),
+              translationHelper.getTranslation(TranslationHelper.TABLE_STOPS,
+                      TranslationHelper.STOP_NAME, base.getId().getId(),
```

Comment:
Are the constants `STOP_NAME` and `TABLE_STOPS ` exposed from the OBA classes somewhere. Feels like a hack to duplicate it here.